libfreerdp/CMakeLists.txt

```diff
@@ -249,6 +249,10 @@ if(WIN32 AND WITH_MEDIA_FOUNDATION)
     set(CODEC_SRCS ${CODEC_SRCS} codec/h264_mf.c)
 endif()
 
+if(ANDROID AND WITH_MEDIACODEC)
+    set(CODEC_SRCS ${CODEC_SRCS} codec/h264_mediacodec.c)
```

Comment:
`list(APPEND CODEC_SRCS codec/h264_mediacodec.c)` is the modern `CMake` version.